services/webhook/tests/test_github_checks_client.py

```
@@ -174,3 +174,32 @@ def test_post_check_run_connect_error_propagates(mock_transport_client):
     with patch("httpx.post", side_effect=lambda *a, **kw: client.post(*a, **kw)):
         with pytest.raises(httpx.ConnectError):
             post_check_run("tok", "o", "r", result)
+
+
+def test_post_check_run_truncates_oversize_summary(monkeypatch):
```

Ruff (0.15.20)

[warning] Unused function argument: `monkeypatch`

(ARG001)